public/base-locales/de.json

```diff
@@ -432,18 +432,18 @@
   "profile_error": "Muss eindeutig und gültig sein",
   "team_0": "Neutral",
   "battle_changes": "im Kampf",
-  "battle_changes_poracle": "battle_changes",
+  "battle_changes_poracle": "wird angegriffen",
```

Comment:
that translation is missing in poracle. maybe rename to `in_angriff`, `Kampfänderung`.... 

this command is used as `!poracle valor battle_changes` (feature has to be enabled in config)

Reply:
https://github.com/KartulUdus/PoracleJS/pull/641
!gym everything d100 battle_changes5 - where 5 is 5min cooldown between next notification
!arena alle d100 wird_angegriffen5  (in german)

Reply:
@TurtIeSocks  does this translation needs underline? so `wird_angegriffen` ? Because poracle commands are with underline!?

Reply:
Yes it needs the `_`

Reply:
@ReuschelCGN can you add the `_` between the words here? Then I can merge.

Reply:
changed